unify_idents/engine_parsers/ident/msfragger_3_parser.py

```diff
@@ -182,15 +182,16 @@ def unify(self):
             self.df (pd.DataFrame): unified dataframe
         """
         self.df["Search Engine"] = "msfragger_3_0"
-        self.df["Raw data location"] = str(self.params["Raw data location"])
-        spec_title = (
-            self.df["Raw data location"].str.split(".").str[0].str.split("/").str[-1]
-        )
+        spec_title = re.search(
+            r"(?<=/)([\w_]+)(?=\.)", self.params["Raw data location"]
+        ).group(0)
         self.df["Spectrum Title"] = (
             spec_title
             + "."
             + self.df["Spectrum ID"].astype(str)
             + "."
+            + self.df["Spectrum ID"].astype(str)
+            + "."
```

Comment:
Is this correct? Why do we need to have the ID twice in there?

Reply:
k forget about this one, just saw that it is always twice in all but this one ^^